service/src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -171,6 +171,11 @@ public void initialize() {
             .context(flightBeanBag)
             .addHook(mdcHook)
             .addHook(new TracingHook())
+            .addHook(
+                new WorkspaceActivityLogHooks(
+                    flightBeanBag.getActivityLogDao(),
```

Comment:
Since JobService is a `@Component`, you can `@Autowire` these instead of using the bean bag. The bag is intended for use by flights. The hook itself is a singleton.